### PR TITLE
[DOCS-30] Update README for Vale instructions (grammar linter) (#69)

### PR DESCRIPTION
**Pull request by mjang-cobalt**
As we can't use Grammarly due to security concerns, I've set up an alternative, [Vale](https://docs.errata.ai/vale/about).

In this PR, I've set up a [GrammarLinter.md README](https://github.com/cobalthq/cobalt-product-public-docs/blob/DOCS-30_update_README_vale/GrammarLinter.md) that describes:

- How you can use Vale to lint your markdown files
- How I've configured Vale (I've customized it for Cobalt content)
- What files to copy from this repo (If you want to run Vale on your own repos)

In previous PRs, I've already set up Vale grammar rules in the `styles/` subdirectory.

### CODE REVIEW

File: GrammarLinter.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+- The `StylesPath` points to rules in the `styles/` subdirectory.
+- The `BasedOnStyles` parameter specifies style subdirectories.
+- The `IgnoredScopes` tells Vale to ignore content as described in [Vale Documentation](https://docs.errata.ai/vale/config#ignoredscopes).
+
+### Use Vale in Your Repository
+
+If you want to use this Vale configuration in your repository, copy:
+
+- `.vale.ini`
```
Comment (parabolic) [inline GrammarLinter.md:34]: I do not see any `.vale.ini` checked in this PR. Are we supposed to create an empty one ?

Comment (ericcornelissen) [reply GrammarLinter.md:34]: It already exists on `main`: https://github.com/cobalthq/cobalt-product-public-docs/blob/main/.vale.ini - it looks like this PR is just adding/updating the documentation related to the use of Vale

Comment (parabolic) [reply GrammarLinter.md:34]: Thank you!

File: GrammarLinter.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Vale pulls rules from YAML files in the `styles/` subdirectory. They include grammar rules in the following subdirectories:
+
+- Modified rules from GitLab in the `styles/cobalt/` subdirectory
+- [Google Developer Style Guide](https://github.com/errata-ai/Google) rules, customized for Vale, in the `styles/Google` subdirectory
+- Rules associated with the [write-good](https://github.com/btford/write-good) grammar linter
+
+These rules are a "Work in Progress"; we may overrule/modify them as we use them to review Cobalt content. For example, if you find a common word / acronym that we use, you're
+welcome to add it (with a PR) to our `styles/cobalt/spelling-exceptions.txt` file.
+
+For more information, see the [Vale documenmtation](https://docs.errata.ai/).
```
Comment (parabolic) [inline GrammarLinter.md:22]: `! vale %` :)
```suggestion
For more information, see the [Vale documentation](https://docs.errata.ai/).
```

File: GrammarLinter.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+## English Grammar Linter (Vale)
+
+This repository includes `beta` rules based on the [Vale grammar linter](https://docs.errata.ai/vale). To install Vale on a Mac, run:
```
Comment (ericcornelissen) [inline GrammarLinter.md:3]: The link in this sentence seems to be a dead link, I'm getting a Page Not Found error. I'm not sure what you intend to link to, maybe `https://docs.errata.ai/#vale` (with `#`)?

File: GrammarLinter.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+```
+brew install vale
+```
+
+You can set up Vale with several different IDEs. For this repository,
+I've tested the integration between Vale and VSCode, and described some tips and tricks
```
Comment (ericcornelissen) [inline GrammarLinter.md:10]: Following the grammar style guides, should we replace "I" with "We" here? The same for line 64

```suggestion
We've tested the integration between Vale and VSCode, and described some tips and tricks
```

File: GrammarLinter.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+- `.vale.ini`
+- The `styles/` subdirectory
+
+If you copy Vale configuration files to your repository, and want to "change the rules",
+open a PR against this cobalt-product-public-docs repo.
+
+### Use Vale in Your IDE
+
+You can set up Vale with several different IDEs. For more information, see the
+[Integrations](https://docs.errata.ai) section of the Vale documentation.
```
Comment (ericcornelissen) [inline GrammarLinter.md:45]: We can point directly to the integrations section. Same for line 66

```suggestion
[Integrations](https://docs.errata.ai/#integrations) section of the Vale documentation.
```

Alternatively, you could use a MarkDown feature where you can define a link and use it repeatedly in the document, e.g.:

```
<!-- other text -->

[Vale Integrations] section of the Vale documentation.

<!-- other text, until the end of the document -->

[Vale Integrations]: https://docs.errata.ai/#integrations

File: GrammarLinter.md (hunk 3 above)
Comment (mjang-cobalt) [reply GrammarLinter.md:3]:
```suggestion
This repository includes `beta` rules based on the [Vale grammar linter](https://docs.errata.ai/#vale). To install Vale on a Mac, run:
```

**Comment by mjang-cobalt** [comment]
@ericcornelissen @parabolic if I've addressed your concerns, I'd appreciate your approval.

**Review by ericcornelissen — APPROVED** [review]

**Comment by mjang-cobalt** [comment]
Thanks @ericcornelissen ! 

FYI, if you find more issues, let me know, or submit a PR.

**Pull request merged**